### DISCUSSION TITLE
Inconsistency Booking Customer id and its documentation

### DISCUSSION THREAD

**Discussion by Loca99** [microsoftgraph/msgraph-sdk-dotnet#2531] [Q&A]
Hello,
the API Get Booking Customer:
`await graphClient.Solutions.BookingBusinesses[bookingBusinessId].Customers.GetAsync();`
return customers with Id with this structure **AAQkAGZjYzBkNjI4LTVkYTctNGY1NS05ZGM2LTIwMjE3MzNhNjUxMgAQAIlE7scJWEFBrmvpbJ**

According to the [documentation](https://learn.microsoft.com/en-us/graph/api/bookingcustomer-get), it should be a Guid.

The POST API also does not return a Guid id and at the moment it seems impossible to retrieve the Id.

**I am currently using release 5.54.0**

Does anyone know anything?
Thank you very much

**Comment by andrueastman** [comment]
Thanks for raising this @Loca99 

As this is API related, we may not be best placed to answer this as this repo is mainly intended for SDK related questions. Any chance you can post the same question at the link below as well? https://aka.ms/askGraph

**Reply by Loca99** [reply]
@andrueastman OK, thank you very much.
(https://learn.microsoft.com/en-us/answers/questions/1691784/inconsistency-booking-customer-id-and-its-document)
